portal/src/index.tsx

```diff
@@ -8,12 +8,27 @@ import "core-js/stable";
 import "regenerator-runtime/runtime";
 
 import "normalize.css";
-
 import "./index.scss";
 
 import React from "react";
 import { render } from "react-dom";
+import authgear from "@authgear/web";
 
 import App from "./App";
 
-render(<App />, document.getElementById("react-app-root"));
+fetch("/api/runtime-config.json")
```

Comment:
I think this initialization should be done after first render? We should show loading indicator while initializing.